### PR TITLE
updated vulnerable 6.9.3 node docker file to 10

### PR DESCRIPTION
**Pull request by hiyadav**
#194

### CODE REVIEW

File: node/express/container/Application/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:6.9.3
+FROM node:10
```
Comment (jalajmsft) [inline node/express/container/Application/Dockerfile]: Should we not move to Node 12 which is current LTS.

**Comment by jalajmsft** [comment]
@hiyadav I am assuming that you would have tested these flows

**Comment by bishal-pdMSFT** [comment]
@hiyadav please take this PR forward. It is linked to P0 bug.

**Comment by hiyadav** [comment]
> @hiyadav I am assuming that you would have tested these flows

Yeah

File: node/express/container/Application/Dockerfile (hunk 1 above)
Comment (hiyadav) [reply node/express/container/Application/Dockerfile]: Yeah, but with that a lot of tests started failing, hence to minimize the changes. Picked this one. Also 10 is also LTS, hence we have a years time to change ths images.

**Pull request merged**